src/components/calcite-dropdown/calcite-dropdown.tsx

```diff
@@ -407,7 +407,11 @@ export class CalciteDropdown {
       this.triggers.includes(target) ||
       this.triggers.some((trigger) => trigger.contains(target))
     ) {
-      if (target.nodeName !== "BUTTON" && target.nodeName !== "CALCITE-BUTTON") {
```

Comment:
My advice would be to not modify this as part of this PR> It seems to be out of scope. The fix isn't a feat to support action elements here.

Reply:
Ok. we can leave this code as is but can you create a follow up issue to refactor this code?

We shouldn't have to keep blacklisting elements here when a different button type of element is slotted.

We can discuss how to clean it up in the new issue.